staging/src/k8s.io/kubectl/pkg/util/templates/templater.go

```diff
@@ -45,6 +45,7 @@ func ActsAsRootCommand(cmd *cobra.Command, filters []string, groups ...CommandGr
 		Filtered:      filters,
 	}
 	cmd.SetFlagErrorFunc(templater.FlagErrorFunc())
+	cmd.SilenceUsage = true
```

Comment:
hi, @sallyom do we have a test for this change?

Reply:
@sallyom good point, make sure to add one covering this fix and the one from #82423 in a follow up PR.